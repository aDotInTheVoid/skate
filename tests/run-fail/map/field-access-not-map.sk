fn main() {
    let x = [1, 2, 3];
    let z = x.y;
}