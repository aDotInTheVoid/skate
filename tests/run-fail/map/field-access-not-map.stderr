error: Expected `map`, got `array`
  ┌─ $DD/tests/run-fail/map/field-access-not-map.sk:3:13
  │
3 │     let z = x.y;
  │             ^ Evaluated to `[1, 2, 3]`

